### ISSUE TITLE
make the model available on google drive

### ISSUE THREAD

**Issue by rafaelvalle** [JXingZhao/EGNet#20]
hi! thank you for your work
it will be helpful to our community if you upload your model to google drive instead of baidu given that the download process is much simpler.

**Comment by vandesa003** [comment]
Yes, Google drive is more efficient. @JXingZhao could you please upload it to google drive. Thanks a lot!
